code/modules/paperwork/records/lore/teth.dm

```diff
@@ -1,21 +1,438 @@
+	
+	"It is no easy feat to cut one�s wrist with a sane mind. A strong will is needed with the resolve to make multiple attempts. Only when it is sliced as if one were simply cutting a mere slab of meat, can they achieve this. Only when one yearns for death that much, can they succeed."<br>
+	"I eternally regret not realizing to ask her why she needed that industrial knife. I eternally regret not realizing that she was hiding her wrists with long sleeves. I eternally regret not realizing that she always took extra care to hide them whenever I spoke with her. I eternally regret not realizing that I never heard her cheerful laughter anymore, and that she roamed around the company with lifeless eyes more and more often."<br>
+	"I can�t even imagine how painful it would be to cut your wrist. I found out that the only reason she could manage to do something like that was she cut hers straight through."<br>
+	"We always popped open a bottle of wine every time we succeeded. It became one of our customs. She told us that it only takes one slight misstep to fall into indifferent habit if we get too used to success. Thus, we should always smile, rejoice, and celebrate every little thing in the wake of success. And we should never neglect failure, as success can be born from a pile of failures. When I saw the latticework of scars on her wrist, I was reminded of our wine cabinet. We had used up so many of the bottles; there was only one left. That day, I popped the cork off the last one."}
+
+//Old Lady
+/obj/item/paper/fluff/lore/teth/oldlady
+	name = "O-01-12"
+	info = {"<h2><center>O-01-12</center></h2>	<br>
+	Old Lady	<br>
+	"An Abnormality taking the form of an old and feeble woman. Anytime someone enters its Containment Unit, it will talk to that person."<br>
+
+	"What it tells differs from time to time, however, it will always commence with �Dearie, dearie, would you like to hear a story?�. This story could be a lullaby, an unspecified poem, a nursery rhyme, a common children�s story, or something entirely new.<br>
+	It knows every children�s story and will not stop recounting until the listener finally leaves."<br>
+
+	<h3><center>Excerpt from Counseling Log</center></h3><br>
+	She may be the loneliest of all the Abnormalities. She�s filled with loneliness, and tells her tales to fill that void. Well, every story needs a listener. When someone enters her room, she�ll try to fill that void. Of course, listening to her is a very dangerous thing. She knows every story on Earth and even those that cannot possibly exist. Our minds can�t cope with such a thing. She could fry your brain in an instant.}
+
+//Forsaken Murderer
```

Comment:
Forgot  " at the end of the last lore file

Reply:
Good catch, thank you! Just fixed it.